welcome-message/main.scss

```diff
@@ -104,10 +102,14 @@
 	}
 
 	.n-welcome-banner__heading--minimized {
+		@include oTypographySans('s');
 		display: none;
 		@at-root .n-welcome-banner--collapsed & {
 			display: table-cell;
 		}
+		@include oGridRespondTo('M') {
+			@include oTypographySans('m');
```

Comment:
You could use [`oTypographySansSize('m')`](https://github.com/Financial-Times/o-typography#2-using-the-mixins) here